src/modules/collectionCuration/reducer.ts

```diff
@@ -0,0 +1,159 @@
+import { loadingReducer, LoadingState } from 'decentraland-dapps/dist/modules/loading/reducer'
+import {
+  ApproveCollectionCurationFailureAction,
+  ApproveCollectionCurationRequestAction,
+  ApproveCollectionCurationSuccessAction,
+  APPROVE_COLLECTION_CURATION_FAILURE,
+  APPROVE_COLLECTION_CURATION_REQUEST,
+  APPROVE_COLLECTION_CURATION_SUCCESS,
+  FetchCollectionCurationFailureAction,
+  FetchCollectionCurationRequestAction,
+  FetchCollectionCurationsFailureAction,
+  FetchCollectionCurationsRequestAction,
+  FetchCollectionCurationsSuccessAction,
+  FetchCollectionCurationSuccessAction,
+  FETCH_COLLECTION_CURATIONS_FAILURE,
+  FETCH_COLLECTION_CURATIONS_REQUEST,
+  FETCH_COLLECTION_CURATIONS_SUCCESS,
+  FETCH_COLLECTION_CURATION_FAILURE,
+  FETCH_COLLECTION_CURATION_REQUEST,
+  FETCH_COLLECTION_CURATION_SUCCESS,
+  PushCollectionCurationFailureAction,
+  PushCollectionCurationRequestAction,
+  PushCollectionCurationSuccessAction,
+  PUSH_COLLECTION_CURATION_FAILURE,
+  PUSH_COLLECTION_CURATION_REQUEST,
+  PUSH_COLLECTION_CURATION_SUCCESS,
+  RejectCollectionFailureSuccessAction,
+  RejectCollectionCurationRequestAction,
+  RejectCollectionCurationSuccessAction,
+  REJECT_COLLECTION_CURATION_FAILURE,
+  REJECT_COLLECTION_CURATION_REQUEST,
+  REJECT_COLLECTION_CURATION_SUCCESS
+} from './actions'
+import { CollectionCuration, CurationStatus } from './types'
+
+export type CurationState = {
+  data: Record<string, CollectionCuration>
+  loading: LoadingState
+  error: string | null
+}
+
+export const INITIAL_STATE: CurationState = {
+  data: {},
+  loading: [],
+  error: null
+}
+
+type CurationReducerAction =
+  | FetchCollectionCurationsRequestAction
+  | FetchCollectionCurationsSuccessAction
+  | FetchCollectionCurationsFailureAction
+  | FetchCollectionCurationRequestAction
+  | FetchCollectionCurationSuccessAction
+  | FetchCollectionCurationFailureAction
+  | PushCollectionCurationRequestAction
+  | PushCollectionCurationSuccessAction
+  | PushCollectionCurationFailureAction
+  | ApproveCollectionCurationRequestAction
+  | ApproveCollectionCurationSuccessAction
+  | ApproveCollectionCurationFailureAction
+  | RejectCollectionCurationRequestAction
+  | RejectCollectionCurationSuccessAction
+  | RejectCollectionFailureSuccessAction
+
+export function curationReducer(state: CurationState = INITIAL_STATE, action: CurationReducerAction): CurationState {
```

Comment:
this reducer should maybe be called `collectionCurationReducer` right?

Reply:
Yup, updated to `collectionCurationReducer`!